dotpay/src/main/java/com/adyen/checkout/dotpay/DotpaySpinnerView.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2019 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by arman on 11/6/2019.
+ */
+package com.adyen.checkout.dotpay
+
+import android.content.Context
+import android.util.AttributeSet
+import com.adyen.checkout.components.model.payments.request.DotpayPaymentMethod
+import com.adyen.checkout.issuerlist.IssuerListSpinnerView
+
+class DotpaySpinnerView : IssuerListSpinnerView<DotpayPaymentMethod, DotpayComponent> {
```

Comment:
These `...SpinnerView`s seem to be unused, is that right? Can we remove them?

Reply:
So this was an idea we had initially, idk if/how it's being used (we might get some more info when we add better analytics), but it's two different types of view representation for the same Component, a merchant could choose to display this payment as a RecyclerView or as a Spinner. 
For us in the Drop-in the RecyclerView makes more sense.